sashimi/hardware/cameras/__init__.py

```diff
@@ -0,0 +1,180 @@
+from sashimi.config import read_config
+import time
+import numpy as np
+
+conf = read_config()
+
+
+class BasicCamera:
+    def __init__(self):
+        super().__init__()
+        self.encoding = "utf-8"
+        self.camera_id = conf["camera"]["id"]
+        self.image_height = 2048
+        self.image_width = 2048
+        self.frame_shape: tuple = (1024, 1024)
```

Comment:
since frame_shape is affected by property getters and setters, the python convention is to call it with an underscore (_frame_shape), see e.g. the paragraph [here](https://pep8.org/#designing-for-inheritance)